thavalon-webapp/src/components/gameComponents/missionResults.tsx

```diff
@@ -1,5 +1,176 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { GameSocket, InboundMessage, InboundMessageType } from "../../utils/GameSocket";
+import { GameMessage, GameMessageType, MissionGoingMessage, MissionResultsMessage, Snapshot } from "./constants";
+import "../../styles/gameStyles/missionResults.scss";
+import { OverlayTrigger, Tooltip } from "react-bootstrap";
+
+enum MissionStatus {
+    Pending = "pending",
+    Going = "going",
+    Passed = "passed",
+    Failed = "failed"
```

Comment:
Does this need to a be a string enum, or can we just get away with default ints, if the contained value doesn't matter?

Reply:
String value matters here because for passed/failed I use this value as part of class name, for border coloring